coalib/bears/meta.py

```diff
@@ -0,0 +1,25 @@
+from collections import defaultdict
+
+
+class bearclass(type):
+    """
+    Metaclass for :class:`coalib.bears.Bear.Bear` and therefore all bear
+    classes.
+
+    Pushing bears into the future... ;)
+    """
+
+    # by default a bear class has no aspects
```

Comment:
It might be confusing because I am not able to express my thoughts properly. Please correct me. :smiley: